tests/system/action/chat_group/test_sort.py

```diff
@@ -0,0 +1,120 @@
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class ChatGroupSortActionTest(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.permission_test_models = {
+            "meeting/1": {"enable_chat": True},
+            "chat_group/31": {
+                "meeting_id": 1,
+                "name": "name_loisueb",
+            },
+            "chat_group/32": {
+                "meeting_id": 1,
+                "name": "name_blanumop",
+            },
+        }
+
+    def test_sort_correct(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV", "enable_chat": True},
+                "chat_group/31": {
+                    "meeting_id": 222,
+                    "name": "name_loisueb",
+                },
+                "chat_group/32": {
+                    "meeting_id": 222,
+                    "name": "name_blanumop",
+                },
+            }
+        )
+        response = self.request(
+            "chat_group.sort",
+            {"meeting_id": 222, "chat_group_ids": [32, 31]},
+        )
+        self.assert_status_code(response, 200)
+        model_31 = self.get_model("chat_group/31")
+        assert model_31.get("weight") == 2
+        model_32 = self.get_model("chat_group/32")
+        assert model_32.get("weight") == 1
+
+    def test_sort_not_enabled(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV", "enable_chat": False},
+                "chat_group/31": {
+                    "meeting_id": 222,
+                    "name": "name_loisueb",
+                },
+                "chat_group/32": {
+                    "meeting_id": 222,
+                    "name": "name_blanumop",
+                },
+            }
+        )
+        response = self.request(
+            "chat_group.sort",
+            {"meeting_id": 222, "chat_group_ids": [32, 31]},
+        )
+        self.assert_status_code(response, 400)
+        assert "Chat is not enabled." in response.json["message"]
+
+    def test_sort_missing_model(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV", "enable_chat": True},
+                "chat_group/31": {
+                    "meeting_id": 222,
+                    "name": "name_loisueb",
+                },
+            }
+        )
+        response = self.request(
+            "chat_group.sort",
+            {"meeting_id": 222, "chat_group_ids": [32, 31]},
+        )
+        self.assert_status_code(response, 400)
+        assert "Id 32 not in db_instances." in response.json["message"]
+
+    def test_sort_another_section_db(self) -> None:
```

Comment:
name is not fitting

Reply:
Renamed into: `test_sort_additional_chat_groups_in_meeting`